client/package.json

```diff
@@ -6,46 +6,46 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "build:client": "webpack --config webpack.client.rails.build.config.js",
-    "build:server": "webpack --config webpack.server.rails.build.config.js",
-    "build:production:client": "NODE_ENV=production webpack --config webpack.client.rails.build.config.js",
-    "build:production:server": "NODE_ENV=production webpack --config webpack.server.rails.build.config.js",
-    "build:dev:client": "webpack -w --config webpack.client.rails.build.config.js",
-    "build:dev:server": "webpack -w --config webpack.server.rails.build.config.js",
-    "hot-assets": "babel-node server-rails-hot.js",
+    "build:client": "webpack --config webpack.client.config.js",
+    "build:server": "webpack --config webpack.server.config.js",
+    "build:production:client": "NODE_ENV=production webpack --config webpack.client.config.js",
+    "build:production:server": "NODE_ENV=production webpack --config webpack.server.config.js",
+    "build:dev:client": "webpack -w --config webpack.client.config.js",
+    "build:dev:server": "webpack -w --config webpack.server.config.js",
     "print-phantomjs-version": "phantomjs --version",
     "start-phantomjs": "phantomjs --webdriver=8910",
     "styleguide": "start-storybook -p 9001"
   },
   "dependencies": {
-    "autoprefixer": "6.3.6",
```

Comment:
cssnext includes it and warns about a duplicate if it is added separately. The same autoprefixer config is given to cssnext.